.gitignore

```diff
@@ -127,3 +127,9 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Vagrant
+.vagrant/
+
+#nohup.out
+nohup.out
```

Comment:
The last character in a file need to be a newline character - most text editors have a setting that makes the editor take care of that for you,